src/components/InformationCheckBox/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Checkbox, styled } from '@mui/material';
+import { Typography } from '../Typography';
+import getInfoCheckboxBackgroundColor from './utils';
+
+type InformationCheckBoxProps = {
+  title: string;
+  description: string;
+  variant?: 'info' | 'warning' | 'error';
+  checked: boolean;
+  onChange: (event: React.ChangeEvent, checked: boolean) => void;
+};
+const InformationCheckBox: React.FC<InformationCheckBoxProps> = ({
+  title,
+  description,
+  variant = 'info',
+  checked,
+  onChange,
+}) => {
+  return (
+    <Wrapper variant={variant}>
+      <Row>
```

Comment:
pls just use flex wrapper instead as you can put all your flex values in as props:
```tsx
<FlexWrapper direction="row" justify="flex-start" align="center" gap={theme.spacing(1.25)}>
```